README.md

```diff
@@ -13,6 +13,22 @@ Single-crate workspaces can use it as:
   run: # your `cargo publish` code here
 ```
 
+# Input options
+
+Every argument is optional.
+
+| Input              | Description                                                                                                                       | Default |
+|--------------------|-----------------------------------------------------------------------------------------------------------------------------------|---------|
+| crate-name         | The crate whose API to check for semver                                                                                           |         |
```

Comment:
Perhaps explain _why_ someone might want to set this? Otherwise this description isn't that useful.

It's fine to link to one of the scenarios below, if the description is too long to fit here.